### DISCUSSION TITLE
How do I create sub collection.

### DISCUSSION THREAD

**Discussion by manishonc** [pocketbase/pocketbase#1212] [Q&A] [answered]
Hi guys I need help 
I creating a quiz app 
I need sub collection for MCQ options .
How can I create sub collection in pocketbase

**Comment by ganigeorgiev** [comment, marked as answer]
PocketBase doesn't have "sub/nested" collections and instead to represent data relationships you can make use of the `relation` field. I'm not sure what are your data requirements, but in general you can create 2 collections and have a many->one relationship or similar, eg.:

1. **questions**
    - title (`text`)
    - description (`text`)

2. **answers**
    - question (single `relation` to the **questions** collection)
    - content (`text`)

```js
// create question
const question = await pb.collection("questions").create({ title, description });

// create and attach answers to the question
const answer1 = await pb.collection("answers").create({ question: question.id, content: "..." });
const answer2 = await pb.collection("answers").create({ question: question.id, content: "..." });
```

And then later to fetch all **questions** with their answers, you can make use of the `expand` query parameter by using the indirect expansion, eg.:

```js
pb.collection("questions").getList(1, 50, { expand: "answers(question)" });
```

More info on `expand` you can find in https://pocketbase.io/docs/expanding-relations/

For general information on what fields are available and the different collection types you can check https://pocketbase.io/docs/collections/.

**Reply by manishonc** [reply]
Thanks For replying. I though of that.
But my structure -. 

Question(text)

Option A(text)
Option B (text)
Option C(text)
option D (text)


If I use seperate collections. 1 for Questions and 1 for Options 
It would be difficult to add Question.
I Can use Type(JSON)

But I would have write in Json insted of Text.
